WeakAuras/GenericTrigger.lua

```diff
@@ -2181,6 +2181,11 @@ do
       startTimeCooldown = 0;
     end
 
+    if (startTimeCooldown > (2^31 + 2^30) / 1000) then
```

Comment:
I assume this is just an arbitrary cutoff?

Reply:
Yes.